Source/WebGPU/WebGPU/RenderPipeline.mm

```diff
@@ -445,14 +450,33 @@ static MTLVertexStepFunction stepFunction(WGPUVertexStepMode stepMode)
     });
 }
 
-RenderPipeline::RenderPipeline(id<MTLRenderPipelineState> renderPipelineState, MTLPrimitiveType primitiveType, std::optional<MTLIndexType> indexType, MTLWinding frontFace, MTLCullMode cullMode, id<MTLDepthStencilState> depthStencilState, uint32_t vertexShaderInputBufferCount, Device& device)
+RenderPipeline::RenderPipeline(id<MTLRenderPipelineState> renderPipelineState, MTLPrimitiveType primitiveType, std::optional<MTLIndexType> indexType, MTLWinding frontFace, MTLCullMode cullMode, id<MTLDepthStencilState> depthStencilState, MTLRenderPipelineReflection *reflection, uint32_t vertexShaderInputBufferCount, Device& device)
+    : m_renderPipelineState(renderPipelineState)
+    , m_device(device)
+    , m_primitiveType(primitiveType)
+    , m_indexType(indexType)
+    , m_frontFace(frontFace)
+    , m_cullMode(cullMode)
+    , m_depthStencilState(depthStencilState)
+#if HAVE(METAL_BUFFER_BINDING_REFLECTION)
+    , m_reflection(reflection)
+#endif
+    , m_vertexShaderInputBufferCount(vertexShaderInputBufferCount)
+{
+#if !HAVE(METAL_BUFFER_BINDING_REFLECTION)
+    UNUSED_PARAM(reflection);
+#endif
+}
+
+RenderPipeline::RenderPipeline(id<MTLRenderPipelineState> renderPipelineState, MTLPrimitiveType primitiveType, std::optional<MTLIndexType> indexType, MTLWinding frontFace, MTLCullMode cullMode, id<MTLDepthStencilState> depthStencilState, const PipelineLayout &pipelineLayout, uint32_t vertexShaderInputBufferCount, Device& device)
```

Comment:
This looks almost identical to the previous decl that I went cross-eyed trying to see the different between them.